nodejs/eks/cluster.ts

```diff
@@ -407,7 +407,7 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         const invokeOpts = { parent, async: true };
         const vpc = aws.ec2.getVpc({ default: true }, invokeOpts);
         vpcId = vpc.then(v => v.id);
-        clusterSubnetIds = vpc.then(v => aws.ec2.getSubnetIds({ vpcId: v.id }, invokeOpts)).then(subnets => subnets.ids);
+        clusterSubnetIds = vpc.then(v => aws.ec2.getSubnets({ tags: { vpcId: v.id }}, invokeOpts)).then(subnets => subnets.ids);
```

Comment:
I don't think tags is right? You might need https://www.pulumi.com/registry/packages/aws/api-docs/ec2/getsubnets/#filters_nodejs which in return should be `name: 'vpc-id'` and `values: [v.id]` instead?